subsys/bluetooth/host/audio/mpl_internal.h

```diff
@@ -29,66 +29,66 @@ extern "C" {
 
 
 /* Track segments */
-struct mpl_tseg_t {
+struct mpl_tseg {
```

Comment:
Outside the scope of this PR, but I'd much prefer the name `mpl_track_seg` instead (without proper context `tseg` doesn't provide much information about what it is).